portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -15,27 +15,59 @@
 package com.google.sps.servlets;
 
 import java.util.ArrayList;
+import com.google.sps.data.Comment;
 import com.google.gson.Gson;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
+@WebServlet("/comment-data")
 public class DataServlet extends HttpServlet {
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        response.setContentType("application/json");
+        ArrayList<Comment> coms = new ArrayList<>();
```

Comment:
Although we should try to use the shortest possible name, we don't want to do it at the cost of clarity. The common recommendation is to avoid acronyms, abbreviations, and in-team jargon. Here you can use "comments" rather than "coms". So it is more smooth to read the codes.

Reply:
Got it. I've made that change, thanks!